### DISCUSSION TITLE
Disabling cmdline popup for keymaps

### DISCUSSION THREAD

**Discussion by jtyler76** [folke/noice.nvim#374] [Q&A] [answered]
Hi there,

I'm a new user of Noice and am really loving it. There's just one thing that's bugging me and that I've yet find a solution/configuration for.

I have keymaps such as "\<leader>w" which corresponds to a ":w<CR>" to save my current buffer. When I use that keymap, the Noice popup command line flickers on the screen. There's no functional issue, but the flicker is really annoying.

Does anyone know of some way to preserve the popup command line behavior when I'm actually typing out a command, but to disable it when keymaps are executed?

Thanks!

**Comment by folke** [comment, marked as answer]
You should either use `<CMD>` for those mappings or add silent=true. For more info, check the Neovim docs

**Reply by jtyler76** [reply]
Thank you that solves my issue :)

**Reply by MrRoiz** [reply]
Awesome! thanks! I was wondering how I could avoid that ❤️
